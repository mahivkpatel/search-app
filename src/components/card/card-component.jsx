import React from 'react'
import './card.styles.css'
import { shallow, mount, render } from 'enzyme'
export const Card = (props) => {
  return (
    <div className="card-container" key={props.user.id}>
      <img
        alt="users"
        src={`https://robohash.org/${props.user.id}?set=set2$size=180*180`}
      />
      <h3>{props.user.name}</h3>
      <h4>{props.user.email}</h4>
    </div>
  )
}
