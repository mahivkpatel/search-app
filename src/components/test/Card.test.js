import React from 'react'

import { shallow, mount, render } from 'enzyme'
import { Card } from '../card/card-component'
const user = {
  id: 1,
  name: 'Leanne Graham',
  username: 'Bret',
  email: 'Sincere@april.biz',
  address: {
    street: 'Kulas Light',
    suite: 'Apt. 556',
    city: 'Gwenborough',
    zipcode: '92998-3874',
    geo: {
      lat: '-37.3159',
      lng: '81.1496',
    },
  },
  phone: '1-770-736-8031 x56442',
  website: 'hildegard.org',
  company: {
    name: 'Romaguera-Crona',
    catchPhrase: 'Multi-layered client-server neural-net',
    bs: 'harness real-time e-markets',
  },
}
it('expect render Card component', () => {
  expect(shallow(<Card user={user} key={1} />)).toMatchSnapshot()
})
