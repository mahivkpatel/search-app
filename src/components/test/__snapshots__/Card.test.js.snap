// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect render Card component 1`] = `ShallowWrapper {}`;
